cpp/src/arrow/engine/substrait/function_test.cc

```diff
@@ -177,6 +177,31 @@ TEST(FunctionMapping, ValidCases) {
        {nullptr, int8(), int8()},
        "0",
        int8()},
+      {{kSubstraitArithmeticFunctionsUri, "sign"},
+       {"SILENT", "-1"},
```

Comment:
```suggestion
       {"-1"},
```
`sign` does not have an overflow argument.

Reply:
Changed. Thanks for catching this.